audb/core/info.py

```diff
@@ -32,7 +32,11 @@ def author(
     Returns:
         author(s) of database
 
-    """
+    Example:
+        >>> author('emodb', version='1.1.0')
```

Comment:
Should we update to `1.1.1`?

Reply:
I would do it in another pull request as it does not only affect the changes here.

Reply:
ok